### PR TITLE
Move to commontooling

### PR DESCRIPTION
**Pull request by j616**
Move to static commontooling for dev/CI workflows
Pivotal ticket: https://github.com/bbc/rd-apmm-python-lib-mediatimestamp/pull/60

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -103,3 +104,66 @@ ENV/
...
+
+# Generated version file
+_version.py
+
+# Temporary directories for files copied out of commontooling
+.tmp/
+
+# Extra gitignore lines from Makefile
+Dockerfile.multi
+Dockerfile.multi
```
Comment (philipnbbc) [inline .gitignore:169]: I didn't notice this before in mediatimestamp, but should these be in here (twice)?

File: Dockerfile_multi.j2 [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+CMD ["discover", "tests", "-v"]
+
+{{ macros.flake8(modname) }}
+
+{{ macros.mypy(modname) }}
+
+{{ macros.wheel(modname) }}
+
+{{ macros.docs(modname) }}
+CMD ["--html", "--force", "--skip-errors", "--output-dir", ".", "{{ modname }}"]
```
Comment (philipnbbc) [inline Dockerfile_multi.j2:31]: Same comment as with mediatimestamp.

File: Makefile [hunk 3]
```diff
@@ -1,135 +1,9 @@
-PYTHON3=`which python3`
-PY2DSC=`which py2dsc`
+USE_VERSION_FILE:=TRUE
 
-PY2DSC_PARAMS?=--with-python2=false --with-python3=true
+# Needed here due to gitignore being generated in standalone.mk but Dockerfile.multi being added in docker.mk
+EXTRA_GITIGNORE_LINES+=Dockerfile.multi
```
Comment (philipnbbc) [inline Makefile:4]: Same comment as mediatimestamp

**Review by philipnbbc — APPROVED** [review]
LGTM. That was a lot to get through!

Some inline queries as before for mediatimestamp

File: .gitignore (hunk 1 above)
Comment (j616) [reply .gitignore]: Good spot. Sorted.

File: Makefile (hunk 3 above)
Comment (j616) [reply Makefile]: Good spot. Sorted.

File: Dockerfile_multi.j2 (hunk 2 above)
Comment (j616) [reply Dockerfile_multi.j2:31]: As with mediatimestamp, this is import errors on 3rd party libs only present in docs builds. They don't seem to affect the generated docs.

**Pull request merged**